src/components/hooks/use-user-browser.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react'
+import { browserName, browserVersion } from 'react-device-detect'
+
+const minimum_required_versions = {
```

Comment:
Please move this to the constants.ts file

Reply:
sure :+1: